client/state/selectors/get-reader-followed-tags.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { filter, sortBy } from 'lodash';
+
+/**
+ * Returns all of the reader tags a user is following
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Array}          Reader Tags
+ */
+export default function getReaderFollowedTags( state ) {
+	return state.reader.tags.items
```

Comment:
this needs to be memoized. it's currently returning a new array every time it's called, even if `state.reader.tags.items` has not changed. This leads to a lot of re-renders of the followed tags list.